CMakeLists.txt

```diff
@@ -100,6 +100,13 @@ if(NOT CUKE_DISABLE_GTEST)
 endif()
 
 #
+# cgreen
+#
+if(NOT CUKE_DISABLE_CGREEN)
+    find_package(cgreen QUIET)
```

Comment:
`QUIET` is not supported by `cgreen-config.cmake`, and even if it was all other `find_package` in the project do not use it